### ISSUE TITLE
Drawer not closing properly

### ISSUE THREAD

**Issue by walidvb** [root-two/react-native-drawer#191]
Hey, i'm getting issues with the drawer:

with static positioning: the drawer first is displayed on top of the other views,  and when closed, the content overflows:
![static](https://cloud.githubusercontent.com/assets/810604/16846723/d54ff3ea-49ed-11e6-9d93-fefefbee03e8.gif)

with overlay: initially the drawer is displayed correctly, but once open, it won't close to the end.
![overlay](https://cloud.githubusercontent.com/assets/810604/16846732/df08fc6a-49ed-11e6-91dc-1bd7ea0e933f.gif)

my code is as follows:

```
<Drawer
        type="static"
        content={<CategoriesList {...this.props} />}
        openDrawerOffset={160}
        closedDrawerOffset={0}
        onOpen={() => this.props.toggleFilters(true)}
        onClose={() => this.props.toggleFilters(false)}
        styles={{
          drawer:{
            backgroundColor: 'white',
            marginTop: 60,
            overflow: 'hidden'
          }
        }}
        open={this.props.ui.filters.drawerOpen}
        tweenHandler={Drawer.tweenPresets.parallax}
        tapToClose={true}
      >
        <MenuBar {...this.props} style={{paddingTop: 20}} />
        <PartnerList partners={this.state.partners} style={{
          flex: 1,paddingLeft: 15, paddingRight: 15, marginBottom: sub.isValid() ? 0 : 60,
        }}/>
        <GetPass float/>
      </Drawer>
```

Versions:
`"react-native": "^0.29.0"`
`"react-native-drawer": "^2.2.6"`

[Link to the code](http://github.com/walidvb/passsport-mobile/blob/518290020b8f81da54f402ef77fcf10806da29ee/app/Home.js#L47)

If it makes a difference, this is wrapped within a `Router` of `react-native-router-flux`

Any help/hints greatly appreciated, and thank you for the hard work!

**Comment by aviadhahami** [comment]
:+1: for the `overlay` option

**Comment by walidvb** [comment]
About the first case(the `static` one), it looks like I had to add a background color to the main component, which fixed it. Overlay is still not closing properly, though.

**Comment by zjjt** [comment]
Hi there, i also have the same issue with overlay...It isnt closing properly any updates on that topic might help us out a great deal.

thanks and cheers

**Comment by nemosdo** [comment]
have the same issue with overlay

**Comment by nemosdo** [comment]
it works fine for me when  i  remove the line  'tweenHandler={Drawer.tweenPresets.parallax}' ,but i still don't know why ,hope it works for you @walidvb  @zjjt

**Comment by redflasher** [comment]
yes, when ''tweenHandler={Drawer.tweenPresets.parallax}' was removed drawer work good.
Cause of problem may be wrong parameter for tween (forget about 'drawer.width' or something else)

**Comment by zjjt** [comment]
i didnt check it out recently as i use the drawer component for android,if you think that is the tween handler which creates the issue ill check it out again and get back to you @nemosdo nemosdo...
